### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by SCarcalicea** [firebase/FirebaseUI-Flutter#84] [Q&A] [answered]
I checked out the main branch using HTTPS and signed Google CLA. I made some changes, a new branch and trying to push the changes in order to create a PR. I am getting a 403, unauthorised.

I am using the Github account declared in the CLA.

Any ideas? :)

**Comment by lesnitsky** [comment]
@SCarcalicea are you trying to push to your fork or directly to this repo?

**Reply by SCarcalicea** [reply]
I checked out master branch and created a new branch out of that and then tried to push the changes to the new branch. I did not find any document which explains how to do it and usually this is the procedure :)
Can you please tell me which is the procedure? :)

**Reply by lesnitsky** [reply, marked as answer]
You can find the contribution guide [here](https://github.com/firebase/FirebaseUI-Flutter/blob/main/docs/contributing.md#contribution-process).

**Reply by SCarcalicea** [reply]
Thank you!
